e2e/run.sh

```
@@ -0,0 +1,59 @@
+	curl -fsS -X GET "http://localhost:${PORT}/metrics" | promtool check metrics
+}
+
+check_rate_limit_metric() {
+	promtool query instant "http://localhost:${PROMETHEUS_PORT}" 'github_rate_limit_core_remaining >= 1' |
+		grep -q 'github_rate_limit_core_remaining'
+}
+
+trap dump_debug ERR
+trap cleanup EXIT
+
+cd $SCRIPT_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)